docs/appendices/release-notes/5.9.0.rst

```diff
@@ -12,42 +12,22 @@ Released on 2024-10-09.
     If you are upgrading a cluster, you must be running CrateDB 4.0.2 or higher
     before you upgrade to 5.9.0.
 
+    Tables that were created before CrateDB 4.x will not function with 5.x
+    and must be recreated before moving to 5.x.x.
+
     We recommend that you upgrade to the latest 5.8 release before moving to
     5.9.0.
 
     Before upgrading, you should `back up your data`_.
 
 .. WARNING::
 
-    Due to a BWC issue, a rolling upgrade from 5.8.x to 5.9.0 can lead
-    to problems with queries on ``sys.users`` table.
-
-    Inserting ``0`` value into a column of a :ref:`NUMERIC<type-numeric>`
-    type via PG wire leads to an error in 5.9.0.
-
-    Due to a regression, introduced in 5.8.3, a rolling upgrade from 5.8.x to
-    5.9.0 can lead to problems when writing into a column of the
-    ``OBJECT(IGNORED)`` type with an array sub-column with values of mixed
-    types.
-
-    Due to a BWC issue, a rolling upgrade from 5.8.x to 5.9.1 can lead
-    to problems when running queries as a user created on an older version
-    during a rolling upgrade.
-
-    Please do a rolling upgrade from 5.8.x to 5.9.2 or higher instead.
-
-.. WARNING::
-
-    Tables that were created before CrateDB 4.x will not function with 5.x
-    and must be recreated before moving to 5.x.x.
+    5.9.0 was a testing release not promoted to stable due to `Known
+    issues`_.
+    The first stable version for the 5.9 line is :ref:`version_5.9.2`. Please
+    upgrade to 5.9.2 directly.
```

Comment:
What if this was immediately below the title? Below "Version 5.9.0". Replacing line 7. (Splitting hairs... This was never released.)

Reply:
Moved it above the note